natgeo/mortar
# [feature/MORTAR-275-4] fix duration display bug

To test:
```
checkout feature/MORTAR-275-4
go to: http://10.0.0.153.xip.io:3000/components/promo
change type to video
make sure is a value below the kicker
```
